src/services/web3/transactions/concerns/contract.concern.ts

```diff
@@ -131,6 +131,7 @@ export class ContractConcern extends TransactionConcern {
         action,
         sender,
         simulate,
+        originalError: error.data?.originalError,
```

Comment:
Data is also optional, should it be:
```suggestion
        originalError: error?.data?.originalError,
```